### ISSUE TITLE
Warn before packages get purged

### ISSUE THREAD

**Issue by RafaelKr** [linuxmint/mintupgrade#56]
mintupgrade told me it will remove some orphan packages and I should add the packages I want to keep via preferences. I did this and clicked "Fix", then they got removed. Probably I should have clicked on "Check again" and only after that on "Fix". A warning with the list of packages it will remove would've been nice.

Also it would be much easier if we just could select which packages we want to keep directly in that list.

![image](https://user-images.githubusercontent.com/14234815/195420208-44c69560-07e6-4e82-bf59-05c58784cb42.png)

Lucky I did a snapshot and can reinstall them and restore their configurations.

**Comment by clefebvre** [comment]
This is no longer relevant. Only newly orphaned pkgs are removed now.
